File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.select import Select
with webdriver.Firefox() as driver:
    wait=WebDriverWait(driver,timeout=10)
    driver.get("https://training-support.net/webelements/alerts")
    print("the title of the page is: " ,driver.title)
    promptAlert = driver.find_element(By.XPATH,"//*[@id=\"prompt\"]")
    promptAlert.click()
    promptalert=wait.until(EC.alert_is_present())
    print("The mesage on alert window is: ",promptalert.text)
    promptalert.send_keys("Awesome!")
    promptalert.accept()
    

    result = driver.find_element(By.XPATH,"//*[@id=\"result\"]")
    print("The result after clicking on alert: ",result.text)
    driver.quit()

File: Selenium/Activities/Activity20.java
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity20 {
	public static void main(String args[]) {
		WebDriver driver;
		driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/alerts");
		System.out.println("The title of the page is: "+driver.getTitle());
		
		WebElement promptAlert = driver.findElement(By.xpath("//*[@id=\"prompt\"]"));
		promptAlert.click();
		
		Alert promptalert = driver.switchTo().alert();
		System.out.println("The message on alert window is: "+promptalert.getText() );
		
		promptalert.sendKeys("Awesome!");
		//Thread.sleep(3000);
		promptalert.accept();
		
		WebElement result = driver.findElement(By.xpath("//*[@id=\"result\"]"));
		System.out.println("The result after accepting the alert message: "+result.getText());
		
		driver.quit();
		
	}

}
